components/intake.py

```diff
@@ -164,12 +172,22 @@ def intake(self) -> None:
             0.0 if self.has_note() else self.inject_intake_speed
         )
 
-    def outtake(self) -> None:
+    def backdrive_intake(self) -> None:
         self.direction = self.Direction.BACKWARD
 
-    def inject(self) -> None:
+    def backdrive_injector(self) -> None:
+        self.desired_injector_speed = -self.inject_intake_speed
+
+    def feed_shooter(self) -> None:
         self.desired_injector_speed = self.inject_shoot_speed
 
+    def has_intake_stalled(self) -> bool:
+        return (
+            self.motor.get_velocity().value < self.INTAKE_STALL_VELOCITY
+            and self.direction != self.Direction.STOPPED
```

Comment:
nit: this is our own enum, use the more efficient identity check
```suggestion
            and self.direction is not self.Direction.STOPPED
```